### PR TITLE
Non-forgotten rule since ancient days

### PR DESCRIPTION
**Pull request by Yonle**
A same thing as how do i do https://github.com/termux/x11-packages/pull/733. In this pull, i only patch some of patches.

### CODE REVIEW

**Comment by Yonle** [comment]
The following files require it's attention to replace `/data/data/com.termux/files/usr` as `@TERMUX_PREFIX@`:

libandroid-wordexp/wordexp.c
libhdf5/aarch64/H5lib_settings.c
libhdf5/arm/CMakeCache.txt
libhdf5/arm/H5lib_settings.c
libhdf5/i686/CMakeCache.txt.hdf5.i686
libhdf5/i686/H5lib_settings.c
libhdf5/x86_64/CMakeCache.txt.hdf5.x86_64
libhdf5/x86_64/H5lib_settings.c

File: packages/fastfetch/src-modules-packages.c.patch [hunk 1]
```diff
@@ -8,7 +8,7 @@
 -    uint32_t dpkg = getNumStrings("/var/lib/dpkg/status", "Status: ");
 -
 -    #if __ANDROID__
--        dpkg += getNumStrings("/data/data/com.termux/files/usr/var/lib/dpkg/status", "Status: ");
+-        dpkg += getNumStrings("@TERMUX_PREFIX@/var/lib/dpkg/status", "Status: ");
```
Comment (xtkoba) [inline packages/fastfetch/src-modules-packages.c.patch:11]: This was in the original file. This breaks the clean application of the patch.

File: packages/osm2pgsql/FindLua.cmake.patch [hunk 2]
```diff
@@ -1,4 +1,4 @@
---- /data/data/com.termux/files/home/.termux-build/osm2pgsql/src/cmake/FindLua.cmake	2021-02-03 10:26:41.000000000 -0500
+--- @TERMUX_HOME@/.termux-build/osm2pgsql/src/cmake/FindLua.cmake	2021-02-03 10:26:41.000000000 -0500
```
Comment (xtkoba) [inline packages/osm2pgsql/FindLua.cmake.patch:1]: Unnecessary.

**Comment by Yonle** [comment]
Closing as there's multiple issues.

Later i will revert the commits and start a real patching.

**Comment by xtkoba** [comment]
In fact not necessary at all.

**Comment by Yonle** [comment]
> In fact not necessary at all.

Yep, it's not necessary. Will take a look somehwere, and may gonna open a PR when i found another one again.

**Comment by agnostic-apollo** [comment]
You know what happens to ghosts who open unnecessary pull requests? :D

**Comment by Yonle** [comment]
> You know what happens to ghosts who open unnecessary pull requests? :D

They eat fish.

**Pull request closed without merging**